README.asciidoc

```diff
@@ -0,0 +1,17 @@
+Maven Base Docker Image
+-----------------------
+
+Building
+~~~~~~~~
+
+`docker build -t base-maven .`
+
+Example Usage
+~~~~~
+
+* `cd [workDir]`
+
+* `docker run --privileged=true -v {backtick}pwd{backtick}:/opt/jboss/myproject base-maven mvn clean install -f /opt/jboss/myproject`
```

Comment:
can you try to mount volume like -v `pwd`:/opt/jboss/myproject:z (append :z)? It should change context automatically.